recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -69,6 +69,12 @@ do_install_append () {
     install -m 0644 ${WORKDIR}/aktualizr-secondary.service ${D}${systemd_unitdir}/system/aktualizr-secondary.service
     install -m 0700 -d ${D}${libdir}/sota/conf.d
     install -m 0700 -d ${D}${sysconfdir}/sota/conf.d
+
+    if [ -n "${SOTA_SECONDARY_CONFIG_DIR}" ] && [ -e "${SOTA_SECONDARY_CONFIG_DIR}" ]; then
```

Comment:
Could be helpful to display a warning if the variable is defined but not an actual directory. 

Also better: `-d` instead of `-e`.